dolphinscheduler-common/src/main/resources/common.properties

```diff
@@ -124,4 +124,7 @@ ml.mlflow.preset_repository=https://github.com/apache/dolphinscheduler-mlflow
 ml.mlflow.preset_repository_version="main"
 
 # way to collect applicationId: log(original regex match), aop
-appId.collect: log
\ No newline at end of file
+appId.collect: log
+
+# The default env list will be load by Shell task
+shell.env_source_list=/etc/profile,~/.bash_profile
```

Comment:
I dont't think it's a good idea to set this default execution since it may cause exceptions to EMR cluster environment variables. Such as AWS EMR.

Reply:
Do you have a good idea? If we remove the default, in most of the system the task might not work well, due to missing env.

Reply:
This operation is unnecessary in common public cloud EMR. Such as AWS EMR, Alicloud EMR, TencentCloud EMR. So I think this should be commented out by default.

Reply:
Ok, I got, I remove the default value and add comment.